app/kmqs/main.js

```diff
@@ -16,8 +16,8 @@ function load(u) {
 }
 
 if (config.kissmetricsKey) {
-  load('//i.kissmetrics.com/i.js');
-  load('//doug1izaerwt3.cloudfront.net/' + config.kissmetricsKey + '.1.js')
+  load('https://i.kissmetrics.com/i.js');
```

Comment:
@stefanpenner it is unrelated, sorry I should have done two PR's.  We had to force it to use https:// because of a limitation in Cordova.  If you'd prefer I can put it behind a flag of some sort to force SSL on.